src/core/fetch/common/routes.ts

```diff
@@ -0,0 +1,16 @@
+import { Address } from '../../types/Common';
+import { SupportedChainId } from '../../types/Chains';
+
+export const routes = {
+  meta: '',
+  health: 'health',
+  chains: 'chains',
+  auth: 'auth',
+  dao: 'd',
```

Comment:
if dao route accept common `chainId` and `address`, we should have a same function like below.